src/porepy/viz/exporter.py

```diff
@@ -520,546 +1064,789 @@ def _export_grid(
         else:
             raise ValueError(f"Unknown dimension {dim}")
 
-    def _export_1d(
-        self, gs: Iterable[pp.Grid]
-    ) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    def _export_1d(self, gs: Iterable[pp.Grid]) -> Meshio_Geom:
         """
         Export the geometrical data (point coordinates) and connectivity
         information from the 1d PorePy grids to meshio.
+
+        Parameters:
+            gs (Iterable[pp.Grid]): 1d grids.
+
+        Returns:
+            Meshio_Geom: Points, 1d cells (storing the connectivity),
+            and cell ids in correct meshio format.
         """
 
-        # in 1d we have only one cell type
+        # In 1d each cell is a line
         cell_type = "line"
 
-        # cell connectivity information
-        num_cells = np.sum(np.array([g.num_cells for g in gs]))
-        cell_to_nodes = {cell_type: np.empty((num_cells, 2))}  # type: ignore
-        # cell id map
-        cell_id = {cell_type: np.empty(num_cells, dtype=int)}  # type: ignore
-        cell_pos = 0
+        # Dictionary storing cell->nodes connectivity information
+        cell_to_nodes: Dict[str, np.ndarray] = {cell_type: np.empty((0, 2), dtype=int)}
+
+        # Dictionary collecting all cell ids for each cell type.
+        # Since each cell is a line, the list of cell ids is trivial
+        total_num_cells = np.sum(np.array([g.num_cells for g in gs]))
+        cell_id: Dict[str, List[int]] = {
+            cell_type: np.arange(total_num_cells, dtype=int).tolist()
+        }
 
-        # points
+        # Data structure for storing node coordinates of all 1d grids.
         num_pts = np.sum([g.num_nodes for g in gs])
         meshio_pts = np.empty((num_pts, 3))  # type: ignore
-        pts_pos = 0
 
-        # loop on all the 1d grids
+        # Initialize offset. Required taking into account multiple 1d grids.
```

Comment:
I don't understand this comment

Reply:
updated, please check again.

Reply:
As in "required for/when taking into..."?